rclcpp/test/rclcpp/test_future_return_code.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+
+#include <sstream>
+
+#include "rclcpp/future_return_code.hpp"
+
+TEST(FutureReturnCode, to_string) {
+  rclcpp::FutureReturnCode frc = rclcpp::FutureReturnCode::SUCCESS;
+  EXPECT_EQ(rclcpp::to_string(frc), "SUCCESS (0)");
```

Comment:
Won't block PR on this, but FYI for next PRs we've been using as standard leaving the constant in the `EXPECT_EQ` tests in the left side of the comparison
```suggestion
  EXPECT_EQ("SUCCESS (0)", rclcpp::to_string(frc));
```

Reply:
Yeah, good call.  I try to do that, but forgot here :).